app/src/main/java/org/feature/fox/coffee_counter/di/module/ApiModule.kt

```diff
@@ -54,16 +58,24 @@ object ApiModule {
     @Provides
     fun provideRetrofit(
         baseUrl: String,
+        scalarsConverterFactory: ScalarsConverterFactory,
         convFact: Converter.Factory,
         okHttpClient: OkHttpClient,
     ): Retrofit {
         return Retrofit.Builder()
+            .addConverterFactory(scalarsConverterFactory)
             .addConverterFactory(convFact)
             .baseUrl(baseUrl)
             .client(okHttpClient)
             .build()
     }
 
+    @Singleton
+    @Provides
+    fun providesScalarsConverterFactory(): ScalarsConverterFactory {
+        return ScalarsConverterFactory.create()
+    }
+
```

Comment:
Dont know why we are using a second ConverterFactory, but the gradle file dependency for scalars is missing

Reply:
Ok, resolved, Gradle did not sync my checkout properly